sycl/include/CL/sycl/detail/pi.h

```diff
@@ -1309,6 +1312,12 @@ __SYCL_EXPORT pi_result piEnqueueKernelLaunch(
     const size_t *local_work_size, pi_uint32 num_events_in_wait_list,
     const pi_event *event_wait_list, pi_event *event);
 
+__SYCL_EXPORT pi_result piEnqueueHostKernelLaunch(
```

Comment:
Why is this API needed? Why not just implement `piEnqueueKernelLaunch` specially in the pi_esimd_cpu.cpp?

Reply:
I removed piEnqueueHostKernelLaunch in https://gitlab.devtools.intel.com/icl/dpcpp-cm/-/merge_requests/248.

Reply:
The new API will be removed in next update of this PR.